### PR TITLE
update image tag to digest

### PR DESCRIPTION
**Pull request by zhaozhanqi**
@huiran0826 @anuragthehatter @weliang1 @rbbratta 
since for disconnect cluster need to use 'digest' for image

### CODE REVIEW

File: testdata/networking/aosqe-pod-for-ping.json [hunk 1]
```diff
@@ -10,7 +10,7 @@
   "spec": {
       "containers": [{
         "name": "hello-pod",
-        "image": "aosqe/pod-for-ping",
+        "image": "docker.io/aosqe/pod-for-ping@sha256:f61ca8fd1dd7dfb8d403b66df549ed5a94fc4626d3b19e2530af48aefa75e5ea",
```
Comment (liangxia) [inline testdata/networking/aosqe-pod-for-ping.json]: Prefix it with docker.io will not work in disconnected cluster even if you have mirror the image.

Comment (zhaozhanqi) [reply testdata/networking/aosqe-pod-for-ping.json]: are you sure? I think the `docker.io` is required. @wzheng1  please correct me if I'm wrong

Comment (wzheng1) [reply testdata/networking/aosqe-pod-for-ping.json]: yes, docker.io is required when using imagecontentsourcepolicy in disconencted cluster. We need to add it to image mirror list and create ICSP for it. @xiuwang

Comment (liangxia) [reply testdata/networking/aosqe-pod-for-ping.json]: Ack

File: testdata/networking/net_admin_cap_pod.yaml [hunk 2]
```diff
@@ -7,7 +7,7 @@ metadata:
 spec:
   containers:
   - name: network-pod
-    image: docker.io/bmeng/centos-network
+    image: docker.io/bmeng/centos-network@sha256:48da37205f9b43424e0983d4c5e7e07f77b7ba1504bbe35e2f264c75dcb4cd15
```
Comment (liangxia) [inline testdata/networking/net_admin_cap_pod.yaml]: The personal account should be replaced.

Comment (zhaozhanqi) [reply testdata/networking/net_admin_cap_pod.yaml]: aha. missed this one. thanks for catching this.  updated

**Pull request merged**